frontend/src/scenes/insights/trendsLogic.js

```diff
@@ -141,25 +142,28 @@ export const trendsLogic = kea({
                 if (props.cachedResults && !refresh) {
                     return props.cachedResults
                 }
+                insightLogic.actions.startQuery()
```

Comment:
Also on the Marius's point we could definitely use this message handling on all the insighs